keyboards/theone/h75/ansi_tkl/keymaps/default/keymap.c

```diff
@@ -0,0 +1,108 @@
+    {80,        79,         78,         NO_LED,    NO_LED,     NO_LED,     77,         NO_LED,     NO_LED,     76,         75,         NO_LED,     74,          73,                     72,          71,         NO_LED},
+}, {
+  // LED Index to Physical Position
+    //0         1           2           3            4            5            6           7            8           9             10           11           12                      13          14
+    {15*0,0},   {15*2,0},   {15*3,0},   {15*4,0},    {15*5,0},    {15*7,0},    {15*8,0},   {15*9,0},    {15*10,0},   {15*11,0},   {15*12,0},   {15*13,0},   {15*14,0},             {15*16,0}, {15*16,10},
+    //15        16          17          18           19           20           21          22           23          24            25           26           27          28
+    {15*14,10}, {15*12,10}, {15*11,10}, {15*10,10},  {15*9,10},  {15*8,10},   {15*7,10},  {15*6,10},   {15*5,10},  {15*4,10},    {15*3,10},   {15*2,10},   {15*1,10},  {15*0,10},
+    //29        30          31          32              33          34          35          36          37          38              39          40          41          42          43          44
+    {15*0,20},  {15*2,20},  {15*3,20},  {15*4,20},   {15*5,20},   {15*6,20},   {15*7,20},  {15*8,20},   {15*9,20},  {15*10,20},   {15*11,20},  {15*12,20},  {15*13,30}, {15*14,30}, {15*16,20}, {15*16,30},
+    //45        46          47              48          49          50          51          52          53              54          55          56          57
+    {15*14,30}, {15*12,30}, {15*11,30},  {15*10,30},  {15*9,30},   {15*8,30},  {15*7,30},   {15*6,30},  {15*5,30},    {15*4,30},   {15*3,30},   {15*2,30},  {15*0,30},
+    //58        59          60              61          62          63          64          65          66              67          68          69          70
+    {15*0,40},  {15*2,40},  {15*3,40},   {15*4,40},   {15*5,40},   {15*6,40},  {15*7,40},   {15*8,40},  {15*9,40},    {15*10,40},  {15*11,40}, {15*13,40},   {15*15,40},
+    //71        72          73          74          75              76          77              78          79          80
+    {15*15,50}, {15*14,50}, {15*13,50}, {15*11,50}, {15*10,50},   {15*9,50},    {15*6,50},    {15*2,50},    {15*1,50},   {15*0,50},
+}, {
+  // LED Index to Flag
+    0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04,
+    0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04,
+    0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04,
+    0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04,
+    0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04,
+    0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04,
+    0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04,
+    0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04, 0x04,
+    0x04
+} };
```

Comment:
This does not belong at the keymap level and needs to be migrated to `info.json` file.
```suggestion
```